src/main/drivers/sdcard_standard.h

```diff
@@ -196,6 +196,7 @@ typedef struct sdcardCSD_t {
 #define SDCARD_CSD_V2_TRAILER_OFFSET                     127
 #define SDCARD_CSD_V2_TRAILER_LEN                        1
 
+#define SDCARD_IDLE_TOKEN                       0xFF
```

Comment:
(IMO 0xff - DO held high -  was easier to understand. It is questionable if 0xff is a token)

Reply:
I've just been through the instances of `SDCARD_IDLE_TOKEN` and I think they all do related to the idle token.

Reply:
Don't worry about it. It just difference in interpretations - does SD card send IDLE tokens or does it hold output high until ready, and only then sending TOKEN (starting with bit 0).